source/partials/next_release/_release.md.erb

```diff
@@ -1,41 +1,89 @@
+  Prevents file-descriptor and classloader leaks by ensuring that:
+
+  * classloaders for agent-launcher and agent are closed properly
+  * temporary jar files that are extracted are removed after closing the
+    classloaders
+  * the file watcher from log4j is replaced with a custom implementation
+
+* <%= link_to_commit 'a8d09f30be8947165696083039146a33a06fc5a0', 'Config repo support for parameters and referencing templates from pipelines.' %>
+* <%= link_to_commit '36b675e24c7bc7185048f46499c699f9a49ff8e4', 'Adds support for bitbucket webhook' %>
+* <%= link_to_commit '6320d55722cde007995579d4fe6ec786b6287ea6', 'Adds support for gitlab webhook' %>
+* <%= link_to_commit '2cb4c003353e0b4b421af50680d6001cfa864a1a', 'Go Server session timeout is configurable now ', 3856 %>
+* <%= link_to_commit '4e5eeda6eced3defed84a85761cf0f101945b90c', 'Updated footer links to point to installed version\'s api and docs pages', 3837 %>
+* <%= link_to_commit '4e5eeda6eced3defed84a85761cf0f101945b90c', 'Adds support for proxies' %>
+* <%= link_to_commit 'c7ec496fe8967206a28902e0f94aae033d5c2386', 'Pipeline group names in the dashboard are linked to corresponding pipeline group settings page for an admin or pipeline group admin.' %>
 
-* <%= link_to_commit '1ccb0294265b4baea091c9068d7813fdde5009b4', 'Ensure order of auth_config in config xml is used to derive the order in which plugins are used for authentication.', 3608 %>
-* <%= link_to_commit '36d546e680aa43c31bf29d2a586ca31e57739bde', 'Remove tfs-impl-10' %>
-* <%= link_to_commit 'd7c4d167fc10101b03af4cd6676c806cef244747', 'Paused pipelines shows the pause reason on the history page.', 2784 %>
-* <%= link_to_commit '52ac924aad78ad2795c088199eb4c70f14b8205a', 'Save agent hostname and ip address as part of build history. This information will be displayed on the job details page for all kinds of agents.', 3368 %>
 
 <h4>Docker Images</h4>
+* We have made a few changes to the <%= link_to("GoCD Server", "https://github.com/gocd/docker-gocd-server/blob/master/CHANGELOG.md")%> and the <%= link_to("GoCD Agent", "https://github.com/gocd/docker-gocd-agent/blob/master/CHANGELOG.md")%> Docker images.
+
+<h4>Bug Fixes</h4>
+* <%= link_to_commit 'cef6649637d4de2d7bedb6b00f0fa759cb310fe6', 'Fixes a race condition between rescheduleHungJobs and rescheduleAbandonedBuildIfNecessary', 3891 %>
+* <%= link_to_commit 'c741b36437557a359fdd62d659a2ca5dbc02e5e4', 'Fixes VSM rendering for local pipelines', 3916 %>
+* <%= link_to_commit 'a1f99453ca629c18753a8835e406e621fa1cb560', 'Improves checksums for agent jar(s) download', 3884 %>
+* <%= link_to_commit '4db7e52bed39e539e4c6f71e789bee188e3e5bd2', 'Do not show the settings icon for config repo pipelines on VSM.', 3881 %>
+* <%= link_to_commit '4db7e52bed39e539e4c6f71e789bee188e3e5bd2', 'This will make sure that roles are assigned to users before granting permissions.', 3878 %>
```

Comment:
Change the sentence? This will make sure seems inappropriate on the release notes, don't you think?